ui/app/components/mount-backend/type-form.hbs

```diff
@@ -4,39 +4,30 @@
 ~}}
 
 {{#each (array "generic" "cloud" "infra") as |category|}}
-  <h3 class="title box-radio-header">
+  <Hds::Text::Display class="has-top-padding-m has-bottom-padding-s" @tag="h2" size="400">
     {{capitalize category}}
-  </h3>
-  <div class="box-radio-container">
+  </Hds::Text::Display>
+  <div class="flex row-wrap row-gap-8px column-gap-16px has-bottom-padding-m">
     {{#each (filter-by "category" category this.mountTypes) as |type|}}
-      <BoxRadio
-        @displayName={{type.displayName}}
-        @type={{type.type}}
-        @glyph={{or type.glyph type.type}}
-        @groupValue={{this.selection}}
-        @groupName="mount-type"
-        @onRadioChange={{mut this.selection}}
+      <SelectableCard
+        id={{type.type}}
+        class="has-top-padding-m has-text-centered small-card"
+        @onClick={{fn @setMountType type.type}}
         @disabled={{if type.requiredFeature (not (has-feature type.requiredFeature)) false}}
-        @tooltipMessage={{if
-          (or (eq type.type "transform") (eq type.type "kmip") (eq type.type "keymgmt"))
-          (concat
-            type.displayName
-            " is part of the Advanced Data Protection module, which is not included in your enterprise license."
-          )
-          "This secret engine is not included in your license."
-        }}
```

Comment:
Yep - consulted Yankun and decided to remove because this is an outdated and non-accessible pattern. We added the `Enterprise` badge instead for these backends (details in PR description above)